android/src/main/java/com/stripeterminalreactnative/listener/RNReaderReconnectionListener.kt

```diff
@@ -29,7 +28,11 @@ class RNReaderReconnectionListener(
         }
     }
 
-    override fun onReaderReconnectStarted(reader: Reader, cancelReconnect: Cancelable) {
+    override fun onReaderReconnectStarted(
+        reader: Reader,
+        cancelReconnect: Cancelable,
+        reason: DisconnectReason
+    ) {
         onReaderReconnectStarted(cancelReconnect)
         context.sendEvent(START_READER_RECONNECT.listenerName) {
             putMap("reader", mapFromReader(reader))
```

Comment:
can we add the disconnect reason to the event?

Reply:
added